### DISCUSSION TITLE
Migrating to another server

### DISCUSSION THREAD

**Discussion by ugintl** [kimai/kimai#2955] [On-Premise: Installation & Updates]
I want to migrate from local to online server. What changes do I need to do?

**Comment by kevinpapst** [comment]
Reinstall Kimai, move the database and adapt .env to your new database.
You might need to move plugins, generated invoices, custom templates as well (check var/ and var/data/)
See https://www.kimai.org/documentation/backups.html

**Reply by ugintl** [reply]
Why reinstall? Generally, I move the files and database, change the database details and it is good enough. I do not have ssh or terminal access online.

**Reply by kevinpapst** [reply]
I'd say: get yourself a professional hoster that includes SSH.
Updating such an installation would be a pain even for myself and I know the app for quite a while... 
You might be able to get it up and running, but FTP installations are not officially supported (meaning no help here).

**Reply by ugintl** [reply]
@kevinpapst It is a free hosting. I am using it to show demo to the client. It is pity.
